### PR TITLE
Protractor cleanup for demo shell

### PR DESCRIPTION
**Pull request by DenysVuika**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

- Cleanup all "synthetic" tests in protractor, i.e. those addressing the demo shell implementation rather than real-life scenarios
- cleanup dead code
- remove custom fonts from the demo shell
- remove various Viewer customisations in demo shell
- remove site picker from document list demo (not used)
- cleanup various css styles in demo shell

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: demo-shell/src/app/components/app-layout/app-layout.component.html [hunk 1]
```diff
@@ -23,13 +23,13 @@
...
     <adf-sidenav-layout-navigation>
-        <ng-template let-isMenuMinimized="isMenuMinimized">
+        <ng-template>
             <mat-nav-list class="app-sidenav-linklist">
                 <ng-container *ngFor="let link of links">
                     <ng-container *ngIf="link.children">
                         <mat-list-item (click)="trigger.openMenu()" [attr.data-automation-id]="link.title | translate" class="app-sidenav-link">
                             <mat-icon matListIcon>{{link.icon}}</mat-icon>
-                            <span matLine *ngIf="!isMenuMinimized()">{{link.title | translate }}</span>
+                            <span matLine>{{link.title | translate }}</span>
```
Comment (nikita-web-ua) [inline demo-shell/src/app/components/app-layout/app-layout.component.html:32]:
```suggestion
                            <span matLine>{{ link.title | translate }}</span>
```

File: demo-shell/src/app/components/app-layout/app-layout.component.html [hunk 2]
```diff
@@ -41,14 +41,14 @@
                             routerLinkActive="app-sidenav-link--active" [routerLinkActiveOptions]="{ exact: true }"
                             [attr.data-automation-id]="link.title | translate" class="app-sidenav-link">
                             <mat-icon matListIcon >{{link.icon}}</mat-icon>
-                            <span matLine *ngIf="!isMenuMinimized()">{{link.title | translate }}</span>
+                            <span matLine>{{link.title | translate }}</span>
```
Comment (nikita-web-ua) [inline demo-shell/src/app/components/app-layout/app-layout.component.html:44]:
```suggestion
                            <span matLine>{{ link.title | translate }}</span>
```

File: e2e/content-services/upload/uploader-component.e2e.ts [hunk 3]
```diff
@@ -93,8 +85,7 @@ describe('Upload component', () => {
             for (const node of nodeList) {
                 try {
                     await uploadActions.deleteFileOrFolder(node);
-                } catch (error) {
-                }
+                } catch (error) {}
```
Comment (nikita-web-ua) [inline e2e/content-services/upload/uploader-component.e2e.ts:88]: Is try catch here needed?

File: e2e/core/viewer/viewer-content-services-component.e2e.ts [hunk 4]
```diff
@@ -318,7 +298,7 @@ describe('Content Services Viewer', () => {
 
             await viewerPage.checkZoomInButtonIsNotDisplayed();
             await viewerPage.checkUnknownFormatIsDisplayed();
-            await expect(await viewerPage.getUnknownFormatMessage()).toBe('Couldn\'t load preview. Unknown format.');
+            await expect(await viewerPage.getUnknownFormatMessage()).toBe("Couldn't load preview. Unknown format.");
```
Comment (nikita-web-ua) [inline e2e/core/viewer/viewer-content-services-component.e2e.ts:301]: do we need await before expect?

File: e2e/content-services/upload/uploader-component.e2e.ts (hunk 3 above)
Comment (DenysVuika) [reply e2e/content-services/upload/uploader-component.e2e.ts:88]: the e2e probably handles missing nodes and does not crash on deletion

File: e2e/core/viewer/viewer-content-services-component.e2e.ts (hunk 4 above)
Comment (DenysVuika) [reply e2e/core/viewer/viewer-content-services-component.e2e.ts:301]: it's a separate ESLint rule to introduce, we have 1000+ `await expect` incorrectly used in tests

**Review by nikita-web-ua — APPROVED** [review]

**Pull request merged**